src/Aspire.Hosting.AppHost/Aspire.Hosting.AppHost.csproj

```diff
@@ -32,4 +32,16 @@
     <PackageDownload Include="Microsoft.DeveloperControlPlane.$(BuildOs)-$(BuildArch)" Version="[$(MicrosoftDeveloperControlPlanedarwinamd64PackageVersion)]" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Reference the analyzer to ensure it's built but don't reference its output -->
+    <ProjectReference Include="..\Aspire.Hosting.Analyzers\Aspire.Hosting.Analyzers.csproj" OutputItemType="AnalyzerAssemblyPath" ReferenceOutputAssembly="false" Private="true" />
```

Comment:
Did you happen to `dotnet pack` this project and make sure the package looks right? (No new dependencies, the analyzer is in the right spot, etc.)

Reply:
![image](https://github.com/user-attachments/assets/b79ba34c-e16e-4a40-a443-e52372fcdcd4)